migrations/env.py

```diff
@@ -70,7 +70,8 @@ def run_migrations_online():
     connection = engine.connect()
     context.configure(
                 connection=connection,
-                target_metadata=target_metadata
+                target_metadata=target_metadata,
+                compare_type=True
```

Comment:
What does this do?

Reply:
This was required for the alembic autogenerator for migration scripts pick up the changed column lengths. Without that line, the generated migration script would do nothing.